libr/anal/p/anal_x86_cs.c

```diff
@@ -1013,8 +1044,16 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		{
 			char *src = getarg (&gop, 1, 0, NULL);
 			char *dst = getarg (&gop, 0, 1, "^");
-			esilprintf (op, "%s,%s,$z,zf,=,$p,pf,=,$s,sf,=,$0,cf,=,$0,of,=",
-				src, dst);
+			if (a->bits == 64 && dst[0] == 'e') {
+				char *_dst = strdup (dst);
+				_dst[3] = '\0';				//it's a bit hacky but yolo
```

Comment:
the strlen chk is not needed since getarg will allways add at least ",=" to the dst, and registers with strlen (reg) == 0 are not allowed